.github/python-nosetests.sh

```diff
@@ -6,5 +6,8 @@ set -uex
 sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu/ trusty main universe"
 sudo apt-get update
 sudo apt-get install -y python-mock python-nose
+curl https://bootstrap.pypa.io/pip/2.7/get-pip.py |sudo python2
+pip install enum
```

Comment:
I'd prefer something like this instead if curl-ing a script:
```suggestion
sudo apt-get install -y pip
pip install enum mock nose
```

Reply:
Cann install pip simply by run `apt-get install -y pip` (install pip3), or `apt-get install -y python-pip`, (get following errors about packages dependencies)

Btw, pip officially recommend instal with curl, 
https://pip.pypa.io/en/stable/installing/

```The following packages have unmet dependencies:
62
 python-pip : Depends: python (>= 2.7)
63
              Depends: python (< 2.8)
64
              Depends: python:any (>= 2.7.1-0ubuntu2)
65
              Depends: python-colorama but it is not going to be installed
66
              Depends: python-distlib but it is not going to be installed
67
              Depends: python-requests but it is not going to be installed
68
              Recommends: python-dev-all (>= 2.6) but it is not installable
69
E: Unable to correct problems, you have held broken packages.
70
Error: Process completed with exit code 100.

Reply:
The package `python2-pip` or similar doesn't seem to exist in ubuntu 20.04 LTS